crates/nu-protocol/src/syntax_shape.rs

```diff
@@ -112,6 +115,7 @@ impl SyntaxShape {
             SyntaxShape::Filesize => Type::Filesize,
             SyntaxShape::FullCellPath => Type::Any,
             SyntaxShape::GlobPattern => Type::String,
+            SyntaxShape::Error => Type::Error,
```

Comment:
if we're adding `SyntaxShape::Error`, I'm kind of wondering if it needs to be in other places as well. Here are some places I question.

1. parser.rs parse_shapename?
2. parser.rs parse_type?
3. ty.rs maybe a new Type?
4. ty.rs to_shape?
5. ty.rs fmt?
6. others?

Reply:
The ones in `ty.rs` were already completed, I was pleasantly surprised to discover. I think the ones you mentioned in `parser.rs` have also been dealt with, but I'll keep a lookout for other areas.

Reply:
ok, i see them now except for in parse_shape_name.

Reply:
oh sorry. I think I'm mistaken.